src/main/java/com/datasonnet/plugins/CSVFormatPlugin.java

```diff
@@ -70,9 +83,14 @@ public String write(Value input, Map<String, Object> params) throws IOException
         CsvSchema csvSchema = builder.build();
 
         CsvMapper csvMapper = new CsvMapper();
-        String value = csvMapper.writerFor(JsonNode.class)
-                .with(csvSchema).writeValueAsString(jsonTree);
-        return value;
+        String value = null;
```

Comment:
The whole `final` thing can be done here too; also another good place to pop out a method